Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/README.md

```diff
@@ -75,6 +75,62 @@ Returns the specified asset.
 | Endpoint.OS | string | Endpoint operating system. | 
 | CVE.ID | string | Common Vulnerabilities and Exposures IDs. | 
 
+`nexpose-get-asset-tags`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| asset_id | Asset ID. | Required |
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Nexpose.AssetTag.type | string | Type of asset tag |
+| Nexpose.AssetTag.created_time | string | Timestamp with the tag was created |
+| Nexpose.AssetTag.name | string | The value of the tag |
+| Nexpose.AssetTag.risk_modifier | string | The risk modifier value associated with criticality tag type |
+
+#### Command example
```

Comment:
Please update the Human Readable Output too

Reply:
Resolved in 7e30033.